src/btree/bt_debug.c

```diff
@@ -697,6 +697,28 @@ __wt_debug_cursor_page(void *cursor_arg, const char *ofile)
     return (__wt_debug_page(cursor->session, cbt->btree, cbt->ref, ofile));
 }
 
+/*
+ * __wt_debug_cursor_las --
+ *     Dump the LAS tree given a user cursor.
+ */
+int
+__wt_debug_cursor_las(void *cursor_arg, const char *ofile)
```

Comment:
I am curious if this will dump all of the las from the disk, or only the portion that is cached in-memory ?

Reply:
It will dump everything, on-disk as well as in-memory.

This is actually @sueloverso's change I picked up. But I've hacked it in a dozen times myself, I agree with her it's generally useful. It would be good if we could dump just the records we care about from the LAS, which is another chunk of code I've hacked in at least a dozen times. :)

Reply:
Thanks, this will be very useful. @tetsuo-cpp also made an attempt at writing a debug version for las, not sure if he ended up checking it in. I can merge develop to relevant history and get this functionality in there.

Reply:
If you're running test/format, you might want the changes in the PR for WT-5211 also. Those are ready to be merged once they're reviewed.